common/smoketest/requirements-release.txt

```diff
@@ -6,3 +6,4 @@ azure-keyvault-certificates>=0.0.0b1
 azure-keyvault-keys>=0.0.0b1
 azure-keyvault-secrets>=0.0.0b1
 azure-storage-blob>=0.0.0b1
+azure-template>=0.0.0b1
```

Comment:
It might be good to add a comment above this explaining the dependency isn't necessary for the smoke tests, but instead used to make sure the template release triggers them (so it's not accidentally removed later on when someone sees it isn't used).